bin/redshift/run_mvn_tests_on_aws.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Runs Redshift connector tests.
+#
+# Run setup_aws_redshift.sh as a prerequisite for creating the Redshift instance and viewing
+# required environment variables.
+#
+# Usage:
+# run_mvn_tests_on_aws.sh
+
+set -xeuo pipefail
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
```

Comment:
imagine that you want the script to work even when repo is downloaded as a zip file, not via `git clone`.
we use `$BASH_SOURCE` for navigating around, see other scripts for examples